pkg/services/folder/model.go

```diff
@@ -13,6 +13,7 @@ var ErrDatabaseError = errutil.NewBase(errutil.StatusInternal, "folder.database-
 var ErrInternal = errutil.NewBase(errutil.StatusInternal, "folder.internal")
 var ErrFolderTooDeep = errutil.NewBase(errutil.StatusInternal, "folder.too-deep")
 var ErrCircularReference = errutil.NewBase(errutil.StatusBadRequest, "folder.circular-reference", errutil.WithPublicMessage("Circular reference detected"))
+var ErrTargetRegistrySrvConflict = errutil.NewBase(errutil.StatusBadRequest, "folder.target-registry-srv-conflict")
```

Comment:
I think this should be an internal error, since the user isn't supposed to end up getting this error unless something is _very_ wrong?

Reply:
Yes, that makes sense. I had copied this over from [quota service](https://github.com/grafana/grafana/blob/main/pkg/services/quota/model.go#L17), but it's a different context there.